src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/Runtime/BindingFlagSupport/ConstructorPolicies.cs

```diff
@@ -41,7 +41,7 @@ public sealed override void GetMemberAttributes(ConstructorInfo member, out Meth
             isNewSlot = false;
         }
 
-        public sealed override bool ImplicitlyOverrides(ConstructorInfo baseMember, ConstructorInfo derivedMember) => false;
+        public sealed override bool ImplicitlyOverrides(ConstructorInfo? baseMember, ConstructorInfo? derivedMember) => false;
```

Comment:
Not sure why they became nullable

Reply:
I see why, but for me its better to make the return value of  `GetAccessorMethod(EventInfo e)` method non null (maybe assert the retrieved value of `accessor`)

Reply:
> maybe assert the retrieved value of accessor

How do we know this is a valid invariant in all use cases?

Reply:
By my understanding there is only case we should consider (other cases are just returning false like in above case)
```csharp
public sealed override bool ImplicitlyOverrides(EventInfo baseMember, EventInfo derivedMember)
{
   MethodInfo baseAccessor = GetAccessorMethod(baseMember);
   MethodInfo derivedAccessor = GetAccessorMethod(derivedMember);
   return MemberPolicies<MethodInfo>.Default.ImplicitlyOverrides(baseAccessor, derivedAccessor);
}

private MethodInfo GetAccessorMethod(EventInfo e)
{
    MethodInfo? accessor = e.AddMethod;
    Debug.Assert(accessor != null);
    return accessor;
}
```
Here we can see if GetAccessorMethod() returns null that could cause NRE, that makes me think the method not expected to return null even EventInfo.AddMethod is nullable, so better to Assert that here and return non null

Reply:
I see your point. I'll change it, however imo it can't hurt to be conservative in non-nullability assertions, at least as long as it only impact internals.